lua/rip-substitute/popup-win.lua

```diff
@@ -214,6 +221,7 @@ end
 ---@param searchPrefill string
 function M.openSubstitutionPopup(searchPrefill)
 	local rg = require("rip-substitute.rg-operations")
+	local rgMatches = require("rip-substitute.matches")
```

Comment:
you are importing this twice, once in the function and once at the top of the file

Reply:
oops
removed the unnecessary one